pkg/subctl/cmd/root.go

```diff
@@ -14,6 +14,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
We could fix this once and for all with

    _ "k8s.io/client-go/plugin/pkg/client/auth"

See [the client-go](https://github.com/kubernetes/client-go/tree/master/examples#auth-plugins) example for details.

Reply:
ohh, does that work like that?, I'll do it, Thanks skitt! :)

Reply:
Done, and tested.